Scripts/DCS-BIOS/lib/modules/aircraft_modules/AJS37.lua

```diff
@@ -2,7 +2,7 @@ module("AJS37", package.seeall)
 
 local Module = require("Module")
 
---- @class A_4E_C: Module
+--- @class AJS37: Module
```

Comment:
Good catch on this. Honestly this should probably just go in a separate PR and get merged right away.